CONTRIBUTING.md

```diff
@@ -46,6 +46,9 @@ Please install and use [pylint](https://pylint.org/) to scan any Python code.
 There is a configuration file that defines some overrides,
 and note the [Editorconfig file](.editorconfig) in the project too.
 
+### Licensing
+
+If you add new (source code) files, please add a header with the license to the file. This clarifies how your work is licensed.
```

Comment:
We should say what the head MUST include:
- Legal copyright line
- Author(s) list
- Declaration of licence, indicating how to refer to the full text if not provided in full

Someone who knows more about this should confirm the above but I think that's why we have what we have.